src/core/renderers/webgl/managers/FilterManager.js

```diff
@@ -488,11 +492,32 @@ export default class FilterManager extends WebGLManager
     /**
      * Destroys this Filter Manager.
      *
+     * @param {boolean} contextLost context was lost, do not free shaders
+     *
      */
-    destroy()
+    destroy(contextLost)
```

Comment:
Could you set `destroy(contextLost = false)` and update the doc `@param {boolean} [contextLost=false] context was lost, do not free shaders` to not be implicit about falsifying?